extension/custom.css

```diff
@@ -27,3 +27,18 @@ tt,
 .wiki-wrapper .wiki-history .commit-meta code {
 	font-family: 'Ubuntu Mono', 'Droid Sans Mono', 'Operator Mono', 'Liberation Mono', 'Source Code Pro', Menlo, Consolas, Courier, monospace !important;
 }
+
+td.blob-code.blob-code-addition:before,
+td.blob-code.blob-code-deletion:before {
+	display: inline-block;
+	font-family: 'Ubuntu Mono', 'Droid Sans Mono', 'Operator Mono', 'Liberation Mono', 'Source Code Pro', Menlo, Consolas, Courier, monospace !important;
```

Comment:
Could you please move this to `custom.css` where we have this font stack defined? Add these new selectors and a comment about what they're for. It'll make it easier if we ever update the font stack for monospace/code text.